### PR TITLE
Adds users to articles and comments

### PR DESCRIPTION
**Pull request by gnardini**
## Summary
Adds a user to all articles and comments. Only their owner can edit and delete them.

**[Trello card](https://trello.com/c/WsJKtuQq/27-agregarle-creador-a-los-articulos-y-comentarios)**

## Known issue
On this PR the admin stuff from the other PR shows up on the schema. I'm not sure how to fix that, should I do so?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,9 @@ gem 'rubocop', '~> 0.42.0', require: false
 
 gem 'devise', '~> 4.2.0'
 
+gem 'bootstrap-sass', '~> 3.3.6'
+gem 'scss_lint', require: false
```
Comment (epintos) [inline Gemfile]: This should only go in development and test group

**Comment by epintos** [comment]
@gnardini Its quite common for the schema to get a bit dirty when you change between branches or you work with other people. What I do somethings is running db:migrate or just cleaning that file and generating it again

**Comment by gnardini** [comment]
Schema looks clean now, only new references added :)

File: app/views/comments/_comment.html.slim [hunk 2]
```diff
@@ -6,5 +6,6 @@ p
   strong
     = t 'comments.comment'
   = comment.body
-p
-  = link_to t(:destroy), [comment.article, comment], method: :delete
+- if comment.user_id == current_user.id
```
Comment (epintos) [inline app/views/comments/_comment.html.slim]: Good, you will be using pundit here

Comment (epintos) [inline app/views/comments/_comment.html.slim]: You can also do comment.user == current_user

File: spec/factories/articles.rb [hunk 3]
```diff
@@ -2,5 +2,6 @@
   factory :article do
     title { Faker::Book.title }
     body { Faker::Lorem.paragraph }
+    user { FactoryGirl.create :user }
```
Comment (epintos) [inline spec/factories/articles.rb]: Here you can leave only `user` and it will work because its another factory

**Comment by epintos** [comment]
👍

**Pull request merged**